app/controllers/api/v1/questionnaires_controller.rb

```diff
@@ -0,0 +1,107 @@
+class Api::V1::QuestionnairesController < ApplicationController
+  
+  # Index method returns the list of JSON objects of the questionnaire
+  # GET on /questionnaires
+  def index
+    @questionnaires = Questionnaire.order(:id)
+    render json: @questionnaires, status: :ok and return
+  end
+  
+  # Show method returns the JSON object of questionnaire with id = {:id}
+  # GET on /questionnaires/:id
+  def show
+    begin
+      @questionnaire = Questionnaire.find(params[:id])
+      render json: @questionnaire, status: :ok and return
+    rescue ActiveRecord::RecordNotFound
+      render json: $ERROR_INFO.to_s, status: :not_found and return
+    end
+  end
+  
+  # Create method creates a questionnaire and returns the JSON object of the created questionnaire
+  # POST on /questionnaires
+  # Instructor Id statically defined since implementation of Instructor model is out of scope of E2345.
+  def create
+    begin
+      @questionnaire = Questionnaire.new(questionnaire_params)
+      @questionnaire.display_type = sanitize_display_type(@questionnaire.questionnaire_type)
+      @questionnaire.save!
+      render json: @questionnaire, status: :created and return
+    rescue ActiveRecord::RecordInvalid
+      render json: $ERROR_INFO.to_s, status: :unprocessable_entity
+    end
+  end
+
+  # Destroy method deletes the questionnaire object with id- {:id}
+  # DELETE on /questionnaires/:id
+  def destroy
+    begin
+      @questionnaire = Questionnaire.find(params[:id])
+      @questionnaire.delete
```

Comment:
Could you please display the status code `204` and label it as `no_content` if everything is functioning correctly?

Reply:
Commit e86e810 addresses this review